x-pack/plugins/security_solution/public/common/components/page/index.tsx

```diff
@@ -57,6 +59,10 @@ export const AppGlobalStyle = createGlobalStyle`
     z-index: 9950;
   }
 
+  /** applies a "toggled" button style to the Full Screen button */
```

Comment:
I am wondering  if we just use a EuiButton, we won't have to do that

Reply:
this is ok, per our out-of-band-conversation